src/lib.rs

```diff
@@ -171,16 +171,10 @@ impl Database {
         }
     }
 
-    /// Loads a font file into the `Database`.
-    ///
-    /// Will load all font faces in case of a font collection.
-    #[cfg(feature = "fs")]
-    pub fn load_font_file<P: AsRef<Path>>(&mut self, path: P) -> Result<(), std::io::Error> {
+    /// Backend function used by load_font_file to load font files
+    fn load_fonts_from_file<P: AsRef<Path>>(&mut self, path: P, data : &[u8]) {
```

Comment:
What this function does? Nothing?

Reply:
Remove `<P: AsRef<Path>>` from this function. It should accept just `std::path::PathBuf`.